jsons/parameters.json

```diff
@@ -10362,7 +10362,7 @@
     ],
     "triggers_rerun": true,
     "value_translations": {},
-    "value_type": "None"
+    "value_type": null
```

Comment:
Imo the value type should stay as "None" for consistency to the format of other value types. The problem with the "None" strings was rather in the actually used values .

Reply:
value type "None"/null generally seems wrong to me. If it's always None, then why does the parameter even exist?
In this case here it should be "str", since the prefix is given as a string. The default value should arguably then also rather be an empty string?
The description is also messed up ...

Reply:
@StSchulze fixed everything you mentioned, no idea if this will crash something, I believe not